test/support/test_helpers.ex

```diff
@@ -36,6 +37,27 @@ defmodule RemoteRetro.TestHelpers do
     Map.put(context, :user, user)
   end
 
+  def persist_other_user_for_retro(context) do
+    context = Map.merge(%{other_user: @other_user}, context)
+    %{other_user: other_user} = context
+    user_params = User.build_user_from_oauth(other_user)
+    other_user =
+      User.changeset(%User{}, user_params)
+      |> Repo.insert!
+
+    Map.put(context, :other_user, other_user)
+  end
+
+  def assign_idea(%{other_user: other_user, retro: retro} = context) do
```

Comment:
hmmm... rather than hardcode an action items values here, i think we should alter the existing `persist_idea_for_retro` function such that it can be assigned to different users depending on the needs of the test. That way, we encapsulate idea creation in a single helper that can take dynamic category, body values, assignee, and so on

Reply:
yes, i agree!  will update.